project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/settings/MinimizationPluginState.kt

```diff
@@ -28,8 +28,9 @@ class MinimizationPluginState : BaseState() {
 
     @Property(surroundWithTag = false)
     @XCollection(style = XCollection.Style.v1, elementName = "minimizationTransformations")
-    val minimizationTransformations: MutableList<TransformationDescriptors> = mutableListOf(
+    var minimizationTransformations: MutableList<TransformationDescriptors> = mutableListOf(
         TransformationDescriptors.PATH_RELATIVIZATION,
     )
     var exceptionComparingStrategy by enum<ExceptionComparingStrategy>(ExceptionComparingStrategy.SIMPLE)
+    var propertyCheckerProjectModificationLens by enum<ProjectItemLensDescriptor>(ProjectItemLensDescriptor.FILE_DELETING)
```

Comment:
I think it's better to create it on demand inside algorithms without providing any settings because it's closely connected with the algorithm